packages/react-next/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -447,9 +448,10 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
 
   private _tryFocusPreviousActiveElement = (options: {
     containsFocus: boolean;
+    documentContainsFocus: boolean;
     originalElement: HTMLElement | Window | undefined;
   }) => {
-    if (options && options.containsFocus && this._previousActiveElement) {
+    if (options && options.containsFocus && options.documentContainsFocus && this._previousActiveElement) {
```

Comment:
Let's leave this check out as it might break some scenario that we're not sure of. Now users who supply their own onFocusRestore can choose to check it if they like.